WhatsApp/app/src/main/res/layout/fragment_notifications.xml

```diff
@@ -1,6 +1,564 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/root_layout"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="wrap_content">
+
```

Comment:
![image](https://user-images.githubusercontent.com/55768328/168639449-07d74a2c-3610-4819-b013-b1773846abb1.png)

you can see on the right the warning signs. Fix those please